src/main/java/me/figo/FigoSession.java

```diff
@@ -270,46 +165,9 @@ public LoginSettings getLoginSettings(String countryCode, String bankCode) throw
     public TaskTokenResponse setupNewAccount(String bankCode, String countryCode, String loginName, String pin) throws FigoException, IOException	{
     	return this.queryApi("/rest/accounts", new SetupAccountRequest(bankCode, countryCode, loginName, pin), "POST", TaskTokenResponse.class);
     }
-    
-    /**
-     * Setups an account an starts the initial syncronization directly
-     * @param bankCode
-     * @param countryCode
-     * @param loginName
-     * @param pin
-     * @return TaskStatusResponse
-     */
-    public TaskStatusResponse setupAndSyncAccount(String bankCode, String countryCode, String loginName, String pin) throws FigoException, IOException, FigoPinException, InterruptedException	{
```

Comment:
If for what reasons ever taskStatus.getMessage() is always "Connecting to server..." i have an infinity loop. 
What if the messages change some day? e.g. an other language?

If taskStatus.getMessage() returns null => NPE
So at least write this like 
"Connecting to server…”.equals(taskStatus.getMessage())